setup.cfg

```diff
@@ -42,8 +42,11 @@ exclude=
   build,
   examples/brainstorm,
   docs
-max-line-length=90
-ignore=W291,E303,W391,F403,F401,W503,W1202
+max-line-length=80
+#ignore=W291,E303,W391,F403,F401,W503,W1202
+select = C,E,F,W,B,B950
+ignore = E501,W503,E203
```

Comment:
can you comment what these are?

Reply:
these are the ones recommended by the developer of "black" (essentially matching black's internal rules, so if you blacken a file, you won't get flake8 warnings). The old ones were some rules I ignored in an ad-hoc fashion, but probably not for good reasons.

Reply:
the rules can be found here: https://lintlyci.github.io/Flake8Rules/